rskj-core/src/main/java/co/rsk/peg/Federation.java

```diff
@@ -64,6 +64,10 @@ public List<FederationMember> getMembers() {
         return members;
     }
 
+    public Optional<FederationMember> getMemberByBtcPublicKey(BtcECKey btcPublicKey) {
+        return members.stream().filter(federationMember -> federationMember.getBtcPublicKey().equals(btcPublicKey)).findFirst();
```

Comment:
i think is more accurate to use findAny instead of findFirst.
We should expect to never have two members with same btcPublicKey (that check is being done in BridgeSupport: https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L2111)